### PR TITLE
[ci] make performance tests run in parallel

### PR DESCRIPTION
**Pull request by tosterberg**
## Description ##

Update the performance pipeline to run all tests in parallel, uses 4 instances total, 3x g5.12x and 1x g5.48x. Additionally, shortened the llm container setup timeout, since some models are ready earlier than the original 5 minute timer.

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: tests/integration/lmic_test_builder.py [hunk 1]
```diff
@@ -267,7 +267,7 @@ def log_metrics(self, sequence):
             file = open("llm/metrics.log", "r")
             metrics = re.sub("'", r'"', file.readline())
             command = f'aws cloudwatch put-metric-data --namespace "LMIC_performance_{sequence["engine"]}" ' \
-                      f'--region "us-east-1" --metric-data "{metrics}"'
+                      f'--region "us-east-1" --metric-data \'{metrics}\''
```
Comment (siddvenk) [inline tests/integration/lmic_test_builder.py:270]: should `\'{metrics}\'` be `\"{metrics}\"` ?

Comment (tosterberg) [reply tests/integration/lmic_test_builder.py:270]: The double quotes weren't parsing properly, so we moved it over to single quotes and confirmed that it was working properly for posting the metrics json in cloudwatch. Just needed the escape character since we were in a f'' string.

**Review by siddvenk — APPROVED** [review]

**Review by sindhuvahinis — APPROVED** [review]
